inspirehep/modules/hal/tasks.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from celery import shared_task
+from flask import current_app
+
+from invenio_db import db
+from inspirehep.utils.record import get_value
+from inspirehep.modules.hal.tei import tei_response
+from inspirehep.modules.hal import hal_api
+
+from .models import InspireHalRecords
+
+from celery.utils.log import get_task_logger
+logger = get_task_logger(__name__)
+
+DATE_FORMAT = "%a, %d %b %Y %X %Z"
+
+
+def doc_type_should_be_sent_to_hal(record):
+    """If the record has at least one institution with a halID, return True."""
+    # FIXME: Figure out how to check if institutions are in HAL
+    return False
```

Comment:
Actually the criteria will be simpler. We shall introduce a flag such as `for_hal` in the schema, that the cataloguers will set to True in order to declare that a record has to be sent.